src/main/java/ui/issuepanel/FilterPanel.java

```diff
@@ -210,6 +217,14 @@ private void unindicatePanelLoading(RepoOpenedEvent e) {
         }
     }
 
+    private void indicatePanelLoading() {
+        addPanelLoadingIndication();
+    }
+
+    private void unindicatePanelLoading() {
```

Comment:
better names (to avoid the non-word `unindicate`): `showPanelLoadingIndicator`, `hidePanelLodingIndicator`